CONTRIBUTING.md

```diff
@@ -178,8 +178,7 @@ A problem must have a unique slug. This slug is used as
 
 ### In exercism/\<TRACK_ID\>
 
-* Do the same as when [porting an
-  exercise](https://github.com/exercism/legacy-docs/blob/main/you-can-help/implement-an-exercise-from-specification.md).
+* Do the same as when [porting an exercise][porting and exercise].
```

Comment:
Typo:
```suggestion
* Do the same as when [porting an exercise][porting an exercise].
```
I'm also very wary of spaces in a keyword, I would prefer `porting-an-exercise`. All other links here follow that convention.

Reply:
I prefer to not have the hyphens, as is seen in the formatting.  But if you want to suggest that change, please do.